temporal-sdk/src/main/java/io/temporal/common/converter/DataConverter.java

```diff
@@ -70,6 +70,10 @@
  */
 public interface DataConverter {
 
+  default boolean useInRootNamespace() {
```

Comment:
The Dataconverter interface should not have any reference to springboot

Reply:
The data converter should follow the same principle as all the other per NS beans